src/Kaba.ts

```diff
@@ -274,12 +274,24 @@ export class Kaba
 
         if (Object.keys(this.jsEntries).length)
         {
+            const compilerConfigs: kaba.WebpackBuildConfig[] = [];
+
+            Object.keys(this.jsEntries).forEach((entry: string) => {
+                if (this.buildModern)
+                {
+                    compilerConfigs.push(this.buildWebpackConfig(entry, this.jsEntries[entry], cliConfig, true));
+                }
+            });
+
+            // The log output gets really big with multiple entry files.
+            // Therefore the linted (legacy) entries are added after the modern ones.
+            Object.keys(this.jsEntries).forEach((entry: string) => {
+                compilerConfigs.push(this.buildWebpackConfig(entry, this.jsEntries[entry], cliConfig, false));
+            });
+
             jsConfig = {
                 common: this.buildWebpackCommon(cliConfig),
```

Comment:
You should merge **everything** in a **single** config object **per entry**. Remove common and only pass the `watch` flag. See https://github.com/Becklyn/kaba/pull/104#discussion_r367884956